scss/_patterns_navigation.scss

```diff
@@ -3,35 +3,17 @@
 // Default header styling
 
 @mixin vf-p-navigation {
-  @include navigation-light;
-  @include navigation-dark;
+  @include navigation-default;
   @include navigation-sidebar;
 }
 
-@mixin navigation-dark {
-  .p-navigation,
-  .p-navigation--dark {
-    @include navigation-pattern(
-      $background-color: $color-dark,
-      $border-color: $color-mid-light,
-      $text-color: $color-x-light
-    );
-  }
-}
-
-@mixin navigation-light {
-  .p-navigation--light {
+@mixin navigation-default {
```

Comment:
Do we need default? When we have a single pattern don't we just put the style directly in vf mixin.

Reply:
There's two patterns - default and sidebar?

Reply:
Aw, good point. Sorry, didn't see the sidebar in the diff.